.cljfmt/indents.clj

```diff
@@ -1,2 +1,21 @@
-{metabase.api.collection/foo [[:inner 0]]
- toucan.db/insert!           [[:block 1]]}
+;; https://github.com/weavejester/cljfmt/blob/master/cljfmt/resources/cljfmt/indents/clojure.clj
+{auto-parse                              [[:inner 0]]
+ check                                   [[:inner 0]]
+ ex-info                                 [[:inner 0]]
+ let-404                                 [[:block 1]]
+ perms/set-has-full-permissions-for-set? [[:inner 0]]
+ perms/set-has-full-permissions?         [[:inner 0]]
+ perms/set-has-partial-permissions?      [[:inner 0]]
+ publish-event!                          [[:block 1]]
+ qp.streaming/streaming-response         [[:inner 0]]
+ ;; Do we want this or not?
+ ;revision/revert!                        [[:inner 0]]
+ t2/select                               [[:block 1]]
```

Comment:
`t2/select` [does not have any special indentation rules](https://github.com/camsaul/toucan2/blob/master/src/toucan2/select.clj), it's just a normal function

Reply:
It seems like we format it differently, though. If you run clojure-lsp (which is really cljfmt for this task) it will change indentation in many (most?) cases. I have no strong opinion, though, if we settle on one format and stick to it.